integration/benchmarks/rax/run_benchmark.py

```
@@ -0,0 +1,656 @@
+        try:
+            info_idx = client.execute_command("FT.INFO", "bench_idx")
+            info_dict = {}
+            if isinstance(info_idx, list):
+                for i in range(0, len(info_idx), 2):
+                    k = info_idx[i].decode() if isinstance(info_idx[i], bytes) else str(info_idx[i])
+                    info_dict[k] = info_idx[i+1]
+            indexed_docs = int(info_dict.get("num_docs", 0))
+            if indexed_docs >= num_docs:
+                break
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)